content/introduction/getting-started-with-sdks/index.mdx

```diff
@@ -0,0 +1,485 @@
+
+**API**: Marketplace API
+
+This example shows how you can transition a transaction that has already
+been initiated. The logged-in user must be a party that is allowed to
+perform the transition - either operator, customer, or provider. This
+can be checked in the transaction process visualizer in Console, or from
+the transaction's `process.edn` file.
+
+```jsx
+const sharetribeSdk = require('sharetribe-flex-sdk');
+
+var sdk = sharetribeSdk.createInstance({
+  clientId: '<Client ID>',
+  clientSecret: '<Client secret>',
+  tokenStore: sharetribeSdk.tokenStore.memoryStore(),
+});
+
+sdk
+  .login({ username: '<User email>', password: '<User password>' })
+  .then((_) => {
+    sdk.exchangeToken().then((res) => {
+      var store = sharetribeSdk.tokenStore.memoryStore();
+      store.setToken(res.data);
+
+      var sdkTrusted = sharetribeSdk.createInstance({
+        clientId: '<Client secret>',
+        tokenStore: store,
+      });
+
+      sdkTrusted.transactions
+        .transition({
+          id: '<Transaction ID>',
+          transition: '<Transition name>',
+          params: {},
+        })
+        .then((res) => {
+          // res.data
+        });
+    });
+  });
+```
+
+### Updating metadata in a transaction
+
+**API**: Integration API
+
+Updating metadata for a transaction requires that you use the
+Integration API.
+
+```jsx
+const IntegrationSdk = require('sharetribe-flex-integration-sdk');
+
+var sdk = sharetribeSdk.createInstance({
+  clientId: '<Client ID>',
+  clientSecret: '<Client secret>',
+});
+
+sdk.transactions
+  .updateMetadata({
+    id: '<Transaction ID>',
+    metadata: {
+      '<Key>': '<Value>',
+    },
+  })
+  .then((res) => {
+    // res.data
+  });
+```
+
+### Updating a listing
+
+**API**: Marketplace API
+
+Updating a listing requires that the listing author's credentials are
+used.
+
+```jsx
+const sharetribeSdk = require('sharetribe-flex-sdk');
+
+var sdk = sharetribeSdk.createInstance({
+  clientId: '<Client ID>',
+});
+
+sdk
+  .login({ username: '<User email>', password: '<User password>' })
+  .then((_) => {
+    sdk.ownListings
+      .update({
+        id: '<Listing ID>',
+        description: '<Updated description',
+        privateData: {
+          '<Key>': '<Value>',
+        },
+        publicData: {
+          '<Key>': '<Value>',
+        },
+      })
+      .then((res) => {
+        // res.data
+      });
+  });
+```
+
+## Additional resources
+
+For the complete documentation on Sharetribe's SDKs, visit our Github
+documentation for the
+[Standard SDK](https://sharetribe.github.io/flex-sdk-js/index.html) and
+the
+[Integration SDK](https://sharetribe.github.io/flex-integration-sdk-js/).
+There, you'll find
+
+- Features
+- Configuration options
+- Additional examples
+- How to use the browser or the
+  [API Playground](https://sharetribe.GitHub.io/flex-sdk-js/try-it-in-the-playground.html)
+  (only available for the standard SDK) for easy testing
+
+<Callout type="info">
```

Comment:
Might be worth accompanying this with a "Never expose the client secret in frontend code." warning callout